Source/Visitable/VisitableViewController.swift

```diff
@@ -5,7 +5,7 @@ open class VisitableViewController: UIViewController, Visitable {
     open weak var visitableDelegate: VisitableDelegate?
     open var visitableURL: URL!
 
-    public convenience init(url: URL) {
+    public required convenience init(url: URL) {
```

Comment:
@joemasilotti I'm concerned that making this `required` is a breaking change for all existing `VisitableViewController` subclasses. Can we make this more lightweight do you reckon? For instance, storing a closure in `TurboConfig` that takes a `url` and produces a visitable vc or subclass? Something like 

```swift
    public var defaultViewController: (URL) -> VisitableViewController = { url in
        VisitableViewController(url: url)
    }
```

Reply:
Great call. I just updated this to accept a block instead and removed the `required` keyword.

Reply:
Nice! Thank you!